### DISCUSSION TITLE
May I continue to use the dietaryindex for calculating DII?

### DISCUSSION THREAD

**Discussion by Whitney678** [jamesjiadazhan/dietaryindex#312] [Q&A] [answered]
Hello author, thank you for your R package. I have noticed that Zhe Xu has made improvements to the calculation of DII. May I continue to use the dietaryindex for calculating DII, or must install install Zhe Xu's  improved  dietaryindexNDP package?

**Comment by jamesjiadazhan** [comment, marked as answer]
You can directly use dietaryindex. Please cite the following articles if you do so:

- Jiada James Zhan, Rebecca A Hodge, Anne Dunlop, et al. (2024) 'Dietaryindex: A User-Friendly and Versatile R Package for Standardizing Dietary Pattern Analysis in Epidemiological and Clinical Studies', The American Journal of Clinical Nutrition. doi: 10.1016/j.ajcnut.2024.08.021.
- Zhe Xu. (2024) 'Improving the dietaryindex R Package: A Proposal to Include Additional Components for More Accurate DII Computation in NHANES', The American Journal of Clinical Nutrition. doi: 10.1016/j.ajcnut.2024.10.023.
